Classes/git/PBGitGraphLine.h

```diff
@@ -8,8 +8,8 @@
 
 struct PBGitGraphLine
 {
-	int upper      : 1;
-	int from       : 8;
-	int to         : 8;
-	int colorIndex : 8;
+	int upper       : 1;
+	long from       : 8;
+	long to         : 8;
+	int colorIndex  : 8;
```

Comment:
Confused. What are those `:` for ? Isn't it strange that `int` and `long` both are 8 ?